.github/workflows/pull_request.yml

```diff
@@ -1,4 +1,4 @@
-on: [pull_request]
+on: [pull_request_target]
```

Comment:
From what I understand in the documentation, anything using `pull_request_target` doesn't actually use the code PR'd. Instead it uses the base repository (master in our case). So I don't think this will work.

https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#pull_request_target

Reply:
I don't think there is a fix to the comment issue:
https://github.com/actions/first-interaction/issues/10

Reply:
I think it will use the code of the PR but the workflow of `master`, if I understood it correctly.

Reply:
oh right I see, ok cool lets try it